indico/MaKaC/PDFinterface/conference.py

```diff
@@ -182,48 +183,38 @@ def __init__(self, abstract, tz=None):
 
     @staticmethod
     def _get_track_classification(abstract):
-        status = abstract.getCurrentStatus()
-        if isinstance(status, review.AbstractStatusAccepted):
-            if status.getTrack() is not None:
-                return escape(status.getTrack().getTitle())
+        if abstract.state == AbstractState.accepted:
+            if abstract.accepted_track:
```

Comment:
Except that this check was the other way round -- you *can* have accepted abstracts with no track